content/en/docs/other-guides/kubeflow-behind-proxy.md

```diff
@@ -0,0 +1,69 @@
++++
+title = "Deploying Kubeflow behind a proxy server"
+description = "Instructions to install and configure Kubeflow behind a proxy server"
```

Comment:
Just a small suggestion: 

- Use "instructions for [doing]" instead of "instructions to [do]" in the description

```suggestion
description = "Instructions for installing and configuring Kubeflow behind a proxy server"
```

🤷‍♀️ 

It's maybe more grammatically correct + based on the following:

<img width="900" alt="image" src="https://user-images.githubusercontent.com/19637339/98407289-4ff35700-2067-11eb-9fba-efbcb4f1d10b.png">

Reply:
@RFMVasconcelos Hey, let us know if there'll be any updates. Just following up 😃 Have a great weekend